unit-tests/live/d400/test-hdr.py

```diff
@@ -0,0 +1,638 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+######################################
+# This set of tests is valid for any device that supports the HDR feature #
+######################################
+
+# test:device D400* !D457
+
+import pyrealsense2 as rs
+from rspy import test, log
+import time
+
+# HDR CONFIGURATION TESTS
+with test.closure("HDR Config - default config", "[hdr][live]"):
+    ctx = rs.context()
+    device = test.find_first_device_or_exit()
+    if ctx is not None:
+        devices_list = ctx.query_devices()
```

Comment:
The original test used to perform a HW reset between each test, using `restart_first_device_and_return_depth_sensor`
This was removed, is this not required for this test suite?
@remibettan ?

Reply:
It would be better to perform HW reset between the tests, if they are changing some parameters, as the exposure for ID sequence 1/2.
Not needed for other tests.